docker/cassandra/Dockerfile

```diff
@@ -7,6 +7,10 @@ ARG cassandra_k8s_addons_jar
 
 COPY install-cassandra /tmp/install-cassandra
 
+# Create 'cassandra' user and group
```

Comment:
Lets move the user & group add and the chown back into `install-cassandra.

Reply:
Done.